### PR TITLE
cut v0.39.0

### LINKED ISSUE #2953: release v0.39.0

### PR DESCRIPTION
**Pull request by balopat**
Fixes #2953.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,83 @@
+# v0.39.0 Release - 09/26/2019
+
+*Note*: This release comes with a new config version `v1beta15`.
+        To upgrade your `skaffold.yaml`, use `skaffold fix`. If you don't upgrade, skaffold will auto-upgrade in memory as best it can, and print a warning message.
+        See [deprecation-policy.md](/deprecation-policy.md) for details on what beta means.
+        The env vars `DIGEST`, `DIGEST_HEX` and `DIGEST_ALGO` are now fail in `envTemplate` fields.
```
Comment (priyawadhwa) [inline CHANGELOG.md]: are now fail fast?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,83 @@
...
+        See [deprecation-policy.md](/deprecation-policy.md) for details on what beta means.
+        The env vars `DIGEST`, `DIGEST_HEX` and `DIGEST_ALGO` are now fail in `envTemplate` fields. 
+
+New Features:
+
+* Optional pull secret for Kaniko [#2910](https://github.com/GoogleContainerTools/skaffold/pull/2910)
+* Add Jib-Gradle support for Kotlin buildscripts [#2914](https://github.com/GoogleContainerTools/skaffold/pull/2914)
+* Add graceful termination for custom builders [#2886](https://github.com/GoogleContainerTools/skaffold/pull/2886)
+* Add docs and tutorial for buildpacks [#2879](https://github.com/GoogleContainerTools/skaffold/pull/2879)
+* kustomize build args [#2871](https://github.com/GoogleContainerTools/skaffold/pull/2871)
```
Comment (tejal29) [inline CHANGELOG.md]: Can you grab the release note from this PR? I think i filled in the release note section here which is more in details

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,83 @@
...
+
+Bug Fixes:
+
+* fix flake TestGetSetFileValues [#2936](https://github.com/GoogleContainerTools/skaffold/pull/2936)
+* Fix helm deployer with imageStrategy helm and fix test runner [#2887](https://github.com/GoogleContainerTools/skaffold/pull/2887)
+* Include build args in cache hash generation [#2926](https://github.com/GoogleContainerTools/skaffold/pull/2926)
```
Comment (tejal29) [inline CHANGELOG.md]: Shd we highlight this a little bit more?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,83 @@
...
+* Add `setFiles` to `HelmDeploy.HelmRelease` skaffold config which will be add `--set-files` argument to helm CLI [#2895](https://github.com/GoogleContainerTools/skaffold/pull/2895)
```
Comment (tejal29) [inline CHANGELOG.md]: This is a config change.

File: CHANGELOG.md (hunk 1 above)
Comment (balopat) [reply CHANGELOG.md]: pardon my hunglish :D thanks

File: CHANGELOG.md (hunk 2 above)
Comment (balopat) [reply CHANGELOG.md]: I don't see the release notes on this one

File: CHANGELOG.md (hunk 3 above)
Comment (balopat) [reply CHANGELOG.md]: sounds good.

File: CHANGELOG.md (hunk 4 above)
Comment (balopat) [reply CHANGELOG.md]: agreed, thanks for spotting it!

File: CHANGELOG.md (hunk 2 above)
Comment (tejal29) [reply CHANGELOG.md]: hmm looks like i put release notes on another PR #2895 
This is also a config change can you change this to 
```
You can now pass `buildArgs` to kustomize deployer in `KustomizeDeploy` config.
```

**Pull request merged**